### PR TITLE
SSL verification control for #21

### PR DESCRIPTION
**Pull request by explody**
See the prior PR https://github.com/digitalocean/pynetbox/pull/23  which included unrelated other features.  This is the SSL part with the other stuff removed.  I'll forego re-describing everything here :)

I'm not sure if I covered the testing specifically requested, but I'm happy to add more if desired.

### CODE REVIEW

File: tests/test_api.py [hunk 1]
```diff
@@ -9,17 +9,71 @@
...
+
+# Keys are app names, values are arbitrarily selected endpoints
+# We use dcim and ipam since they have unique app classes
+# and circuits because it does not. We don't add other apps/endpoints
+# beyond 'circuits' as they all use the same code as each other
+endpoints = {
+    'dcim': 'devices',
+    'ipam': 'prefixes',
+    'circuits': 'circuits',
+}
```
Comment (zachmoody) [inline tests/test_api.py]: Just need one more line of separation here.

File: tests/test_api.py [hunk 2]
```diff
@@ -9,17 +9,71 @@
...
+
+    def test_ssl_verify_false(self):
+        kwargs = dict(def_kwargs, **{'ssl_verify': False})
+        self.common_arguments(kwargs, 'ssl_verify', False)
+
+    def test_ssl_verify_string(self):
+        kwargs = dict(def_kwargs, **{'ssl_verify': '/path/to/bundle'})
+        self.common_arguments(kwargs, 'ssl_verify', '/path/to/bundle')
+
+
```
Comment (zachmoody) [inline tests/test_api.py]: And one less line at EOF.

**Comment by WatchMeCalculate** [comment]
Can anything be done for the MR?  It seems OP hasn't been active in several months, and it would be nice to have this merged.

**Comment by LukeRussell** [comment]
I'd love to see this merged as well.

My scripts all start with this hack:
```
from functools import partial
requests.get = partial(requests.get, verify=False)
```

**Pull request merged**